test/tests/12-edit-chroot

```diff
@@ -88,6 +83,16 @@ host enter-chroot -n "$release" true
 # Verify croutonversion outputs correct information
 host edit-chroot -l $release | tee /dev/stderr | passes grep "^release: $release"
 
+# Back up with a split
+host edit-chroot -y -s 50000000 -f "$BACKUPDIR/$release.tar" -b "$release"
+test "`ls "$BACKUPDIR"/* | wc -l`" -ge 3
+du -b "$BACKUPDIR"/* | passes awk '$1>50000000{exit 1}'
+
+# Restore the split archive. Fails first due to existence
+fails host edit-chroot -y -f "$BACKUPDIR/$release.tar" -r "$release"
+host edit-chroot -y -f "$BACKUPDIR/$release.tar" -rr "$release"
+host enter-chroot -n "$release" true
+
```

Comment:
Might be good to add a test for compressed split archives, since this was broken before.

Using
```
test `"cat archive.tar.gz | hexdump -v -n2 -e'"%02x"'`" = "8b1f"
```
To check that the file is indeed compressed (`8b1f` is for gzip).